sonar-db/src/test/java/org/sonar/db/ce/CeActivityDaoTest.java

```diff
@@ -146,6 +147,24 @@ public void test_countByQuery() {
   }
 
   @Test
+  public void selectByQuery_no_results_if_shortcircuited_by_component_uuids() {
+    insert("TASK_1", REPORT, "PROJECT_1", CeActivityDto.Status.SUCCESS);
+
+    CeActivityQuery query = new CeActivityQuery();
+    query.setComponentUuids(Collections.<String>emptyList());
+    assertThat(underTest.selectByQuery(db.getSession(), query, new RowBounds(0, 10))).isEmpty();
```

Comment:
I don't ask to modify anything here. But at the DAO level, we should now pass the https://github.com/SonarSource/sonarqube/blob/82564709cf06447b8a7e5fcadc609d892fc85509/sonar-plugin-api/src/main/java/org/sonar/api/utils/Paging.java#L26-26 and add a method in `DbUtils#pagingToRowBounds`